### PR TITLE
skip Sentry for some exception types

### PR DESCRIPTION
**Pull request by aub**
For some exception types, there's no point in sending the data to Sentry because the exceptions do not indicate any actionable event. This adds a filter for ignoring them.

For reviewers, do you think we should also stop logging the errors as well?

### CODE REVIEW

**Comment by carlmjohnson** [comment]
I was thinking about this before in relation to the 403, and my instinct was to add a class variable like "LOG_TO_SENTRY = true" and check for that. I think it makes sense to have the info as part of the class. But maybe you think it's better to have a list of exceptions near the logger?

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -52,6 +59,9 @@ def routing_error
   end
 
   def log_error(exception)
+    unless SKIP_SENTRY_EXCEPTION_TYPES.include?(exception.class)
+      Raven.capture_exception(exception) if ENV['SENTRY_DSN'].present?
+    end
     Raven.capture_exception(exception) if ENV['SENTRY_DSN'].present?
```
Comment (austin-martinez) [inline app/controllers/application_controller.rb]: should this line be removed?

**Comment by aub** [comment]
@carlmjohnson I thought about that too, but it feels like this way we keep the knowledge of Sentry even existing in just one place. I don't feel strongly about it at all though, and I think there's arguments either way.

**Comment by saneshark** [comment]
There is a "type" attribute I added to the MinorCodes module that Common::Exceptions uses, those were meant for possibly generating the Common::Exception classes using meta programming in a future refactor. They could in theory be repurposed to tell whether or not sentry should be used.

My plan now is to move MinorCodes into an i18n yaml file instead in a future refactor.

**Comment by carlmjohnson** [comment]
I think this is fine. I just felt like the alternatives needed to be discussed so we made an informed choice. I think for our use case of having relatively few exceptions to exclude using a skip list works. We can look into using the type attribute as part of a future refactoring.

**Comment by saneshark** [comment]
Agree LGTM

**Comment by ayaleloehr** [comment]
Is the action with @aub to fix the conflict then we're good to merge?

**Pull request merged**